src/api/python/_api.py

```
@@ -622,6 +622,20 @@ def check_xfer_state(self, handle: nixl_xfer_handle) -> str:
         else:
             return "ERR"
 
+    def parallel_transfer(self, handles: list, num_threads: int = 0):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `parallel_transfer`

Add return type annotation: `None`

(ANN202)